variables.tf

```diff
@@ -66,4 +66,9 @@ variable "managed_identity_object_id" {
 variable "palo_cluster_size" {
   type        = "string"
   default     = "1"
+}
+
+variable "queue_secret_count" {
+  type        = "string"
+  default     = "0"
```

Comment:
I think it would be best to make it a local variable (staging only applies to aat) and call it `staging-resource-count`, as it shouldn't be just about queue secrets, but any staging-specific resources.